rollup.config.js

```diff
@@ -1,146 +1,59 @@
-/* eslint-env es6 */
+import resolve from '@rollup/plugin-node-resolve';
+import cjs from "rollup-plugin-cjs-es";
+import buble from '@rollup/plugin-buble';
+import { string } from "rollup-plugin-string";
```

Comment:
What's the difference between this and `require('./rollup.plugins').stylesheet` that was used before?

Reply:
Hmm. It looks like it no longer extracts the CSS files that were produced in `dist/` when running `gulp build` before